metabase/metabase
# funnel chart handle duplicated steps

Closes https://github.com/metabase/metabase/issues/45296

### Description

Handles duplicated funnel steps by summing their values.

### How to verify

```sql
select 'foo' x, 100 y
union all select 'foo', 90
union all select 'bar', 80
union all select 'baz', 70
union all select 'baz', 10
```

- create a funnel chart from the query above
- ensure the funnel chart does not show duplicated steps and instead sums their values

### Checklist
